TokenTrackerLinux/scripts/bundle-node-linux.sh

```
@@ -0,0 +1,101 @@
+if [[ -z "$EXPECTED_SUM" || "$EXPECTED_SUM" != "$ACTUAL_SUM" ]]; then
+  echo "Node.js checksum mismatch for $NODE_TAR" >&2
+  echo "expected: $EXPECTED_SUM" >&2
+  echo "actual:   $ACTUAL_SUM" >&2
+  exit 1
+fi
+
+tar -xzf "$TMPDIR_BUNDLE/$NODE_TAR" -C "$TMPDIR_BUNDLE" "node-v${NODE_VERSION}-${NODE_PLATFORM}/bin/node"
+cp "$TMPDIR_BUNDLE/node-v${NODE_VERSION}-${NODE_PLATFORM}/bin/node" "$EMBED_DIR/node"
+chmod +x "$EMBED_DIR/node"
+
+BUNDLED_NODE_VERSION="$($EMBED_DIR/node -p 'process.versions.node')"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)